_includes/quick_start_local.html

```diff
@@ -58,9 +58,6 @@
       <div class="col-md-12 title-block mobile-heading">
         <div class="option-text">Package</div>
       </div>
-      <div class="col-md-3 option block" id="conda">
-        <div class="option-text">Conda</div>
-      </div>
       <div class="col-md-3 option block selected" id="pip">
```

Comment:
Please use ``col-md-4`` for pip, libtorch and source this should show the row correctly